src/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -33,20 +27,11 @@ public sealed partial class Pipe
         public System.IO.Pipelines.PipeWriter Writer { get { throw null; } }
         public void Reset() { }
     }
-    public partial struct PipeAwaiter<T> : System.Runtime.CompilerServices.ICriticalNotifyCompletion, System.Runtime.CompilerServices.INotifyCompletion
-    {
-        private object _dummy;
-        public PipeAwaiter(System.IO.Pipelines.IPipeAwaiter<T> awaiter) { throw null; }
-        public bool IsCompleted { get { throw null; } }
-        public System.IO.Pipelines.PipeAwaiter<T> GetAwaiter() { throw null; }
-        public T GetResult() { throw null; }
-        public void OnCompleted(System.Action continuation) { }
-        public void UnsafeOnCompleted(System.Action continuation) { }
-    }
     public partial class PipeOptions
     {
-        public PipeOptions(System.Buffers.MemoryPool<byte> pool = null, System.IO.Pipelines.PipeScheduler readerScheduler = null, System.IO.Pipelines.PipeScheduler writerScheduler = null, long pauseWriterThreshold = (long)0, long resumeWriterThreshold = (long)0, int minimumSegmentSize = 2048) { }
+        public PipeOptions(System.Buffers.MemoryPool<byte> pool = null, System.IO.Pipelines.PipeScheduler readerScheduler = null, System.IO.Pipelines.PipeScheduler writerScheduler = null, long pauseWriterThreshold = (long)0, long resumeWriterThreshold = (long)0, int minimumSegmentSize = 2048, bool useSynchronizationContext = true) { }
         public static System.IO.Pipelines.PipeOptions Default { get { throw null; } }
+        public bool UseSynchronizationContext { get { throw null; } }
```

Comment:
Run `msbuild /t:GenerateReferenceSource` to help generate the ref file.

For example:
```text
D:\GitHub\Fork\corefx\src\System.IO.Pipelines\src> msbuild /t:GenerateReferenceSource
```
This will alphabetically sort this property to be where it should (i.e. on line 39 instead of here).
![image](https://user-images.githubusercontent.com/6527137/36939770-cf8af0a2-1eeb-11e8-9a60-01598418dcfb.png)